src/components/ButtonPanel.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Button from './Button';
+
+export default function ButtonPanel() {
+  const buttons = [['AC', '+/-', '%', '÷'],
+    ['7', '8', '9', 'X'],
+    ['4', '5', '6', '-'],
+    ['1', '2', '3', '+'],
+    ['0', '.', '='],
+  ];
+
+  return (
+    <div>
+      {buttons.map((e, index) => (
+        <div key={`group-${index}`}>
+          {e.map(name => <Button key={name} name={name} />)}
+        </div>
+      ))}
+    </div>
+  );
+}
```

Comment:
I think using a random number is better than using the index of the array hence the linter warnings. 

Alternatively, you could also use a module that generates a random string, see https://www.npmjs.com/package/uuid